gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/replication/ConfigBasedDatasetsFinderTest.java

```diff
@@ -99,9 +99,9 @@ public void blacklistPatternTest() {
     patternList.add(pattern1);
     patternList.add(pattern2);
 
-    Assert.assertFalse(configBasedMultiDatasets.blacklistFilteringHelper(configBasedDataset, Optional.of(patternList)));
-    Assert.assertFalse(configBasedMultiDatasets.blacklistFilteringHelper(configBasedDataset2, Optional.of(patternList)));
-    Assert.assertTrue(configBasedMultiDatasets.blacklistFilteringHelper(configBasedDataset3, Optional.of(patternList)));
+    Assert.assertFalse(configBasedMultiDatasets.isDatasetSurviveBlacklistFiltering(configBasedDataset, Optional.of(patternList)));
```

Comment:
change method name from isDatasetSurviveBlacklistFiltering to doesDatasetSurviveBlacklistFiltering?